agent/lib/kontena/workers/weave_worker.rb

```diff
@@ -48,19 +48,21 @@ def on_dns_add(topic, event)
     # @param [String] topic
     # @param [Docker::Event] event
     def on_container_event(topic, event)
-      if event.status == 'start'
+      if network_adapter.router_image?(event.from)
```

Comment:
This actor sync call will now be made for each container event, could it exhaust celluloid?